tests/TestHelpers/GraphHelper.php

```diff
@@ -586,7 +586,7 @@ public static function addUsersToGroup(
 	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
-	public static function addGroupToGroup(
+	public static function triesToaddGroupToGroupthroughPatchRequest(
```

Comment:
IMO may be old naming is good enough since we specify this in a parent function that user has tried to add group to another group.

Reply:
@SagarGi sagar author suggested me this